examples/semantic_search/cohere_wikipedia_search.ipynb

```diff
@@ -0,0 +1,516 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "55683ea8",
+   "metadata": {},
+   "source": [
+    "# Building a RAG chain from Wikipedia\n",
+    "\n",
+    "This notebook shows how to use ApertureDB as part of a Retrieval-Augmented Generation Langchain pipeline.  This means that we're going to use ApertureDB as a vector-based search engine to find documents that match the query and then use a large-language model to generate an answer based on those documents. \n",
+    "\n",
+    "We already have a corpus of >600k paragraphs from the Simple English Wikipedia with associated embeddings provided by Cohere.\n",
+    "(If not, see [Ingesting Wikipedia into ApertureDB](./cohere_wikipedia_ingest.ipynb)).\n",
+    "We'll use that to answer natural-language questions.\n",
+    "\n",
+    "![RAG workflow](images/RAG_Demo.png)\n",
```

Comment:
Ok a few suggestions on the image to make it tie better to what we need:
1. Either put logo or explicitly say ApertureDB vector store and document store
2. swap the blue and green - Blue is our color. green is all outside stuff. 
3. YOu can also name the models you have used  Embedding model e.g. Cohere xyz , LLM e.g. openai gpt4o 

Some optional suggestions: maybe lose the 3d ness and straighten / align arrows

Reply:
Vishakha will take on improvements to this image.